### PR TITLE
Smorgasbord of very small things

### PR DESCRIPTION
**Pull request by andrewconner**
Tagging relevant people in the code

### CODE REVIEW

File: kifi-backend/modules/eliza/app/com/keepit/eliza/util/MessageFormatter.scala [hunk 1]
```diff
@@ -35,7 +35,7 @@ object MessageFormatter {
       kind match {
         case "i" => ImageLookHereSegment(text, payload)
         case "r" => TextLookHereSegment(text, payload)
-        case _ => throw new Exception("Unknown look-here type: " + kind)
+        case _ => throw new Exception(s"Unknown look-here type in ${m.matched}")
```
Comment (andrewconner) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/util/MessageFormatter.scala:38]: We have some look heres that are failing to parse correctly. Dumping out more info so we can debug. @eishay

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/OrganizationInviteCommander.scala [hunk 2]
```diff
@@ -434,8 +434,9 @@ class OrganizationInviteCommanderImpl @Inject() (db: Database,
 
         val basicUserById = db.readOnlyReplica { implicit session => basicUserRepo.loadAllActive(nonMembers.toSet) }
 
-        val suggestedUsers = nonMembers.map { userId =>
-          MaybeOrganizationMember(Left(basicUserById(userId)), OrganizationRole.MEMBER, lastInvitedAt = None)
+        val suggestedUsers = nonMembers.collect {
+          case uid if basicUserById.get(uid).isDefined =>
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/OrganizationInviteCommander.scala:438]: If inactive users get suggested, they blow this up. Just skip them.

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/TwitterSyncCommander.scala [hunk 3]
```diff
@@ -124,14 +124,17 @@ class TwitterSyncCommander @Inject() (
...
+              // Grab Twitter SUIs, prefer ones that are "fetched_using_self", which is the state for active, working records
+              socialRepo.getByUser(userId).filter(s => s.networkType == SocialNetworks.TWITTER).sortBy(sui => sui.state == SocialUserInfoStates.FETCHED_USING_SELF).reverse.headOption
             }
             val library = libraryRepo.get(state.libraryId)
 
             (socialUserInfo, library)
           }
 
-          if (library.state == LibraryStates.ACTIVE) {
+          if (socialUserInfo.exists(_.state == SocialUserInfoStates.TOKEN_EXPIRED)) {
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/TwitterSyncCommander.scala:135]: @JRuff42 @stkem Fixes some exceptions where token was revoked.

File: packrat/main.js [hunk 4]
```diff
@@ -424,6 +424,17 @@ var socketHandlers = {
       api.requestUpdateCheck();
     }
   },
+  flush: function (t) {
```
Comment (andrewconner) [inline packrat/main.js:427]: @cvializ Simple flushing mechanism. Server can flush clients caches, like when they join orgs.

Comment (cvializ) [inline packrat/main.js:427]: awesome! hopefully this should keep things in sync

**Pull request merged**